app/bootstrap.php

```diff
@@ -63,14 +64,15 @@ function (ConsoleCommandEvent $event) use ($userInteraction) {
 };
 
 $container['installer.docker'] = function ($c) {
+    $dinghyCli = new DinghyCli($c['process.interactive_runner']);
```

Comment:
Isn't it more relevant to move it to a container service if it's used in more than one place ?